src/contrib/cluster/Akka.Cluster.Sharding/Serialization/ClusterShardingMessageSerializer.cs

```diff
@@ -224,6 +241,11 @@ public override string Manifest(object o)
                 case ShardRegionStats _: return ShardRegionStatsManifest;
                 case GetClusterShardingStats _: return GetClusterShardingStatsManifest;
                 case ClusterShardingStats _: return ClusterShardingStatsManifest;
+                case GetCurrentRegions _: return GetCurrentRegionsManifest;
+                case CurrentRegions _: return CurrentRegionsManifest;
+                case GetShardRegionState _: return GetShardRegionStateManifest;
+                case ShardState _: return ShardStateManifest;
```

Comment:
Oh boy, did we have a namespace overload here? I.e. `ShardState` and `Shard.ShardState`?

Reply:
Looks like it